shelley/chain-and-ledger/shelley-spec-ledger-test/test/Test/Shelley/Spec/Ledger/PropertyTests.hs

```diff
@@ -55,7 +56,8 @@ minimalPropertyTests =
         "Deserialize stake address reference"
         [ TQC.testProperty "wstake reference from bytestrings" propDeserializeAddrStakeReference,
           TQC.testProperty "stake reference from short bytestring" propDeserializeAddrStakeReferenceShortIncrediblyLongName
-        ]
+        ],
+      TQC.testProperty "legacy overlay schedule" legacyOverlayTest
```

Comment:
@nc6 I've added a property test here that checks that the new and old overlay schedules agree. I've added it to the minimal property tests, but it is borderline too long (it's about a minute on my laptop). we can move it to the nightly tests if that's better.